packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -84,472 +62,732 @@ describeNoCompat("GC DataStore Tombstoned When It Is Sweep Ready", (getTestObjec
         }
     });
 
-    // This function creates an unreferenced datastore and returns the datastore's id and the summary version that
-    // datastore was unreferenced in.
-    const summarizationWithUnreferencedDataStoreAfterTime =
-    async (approximateUnreferenceTimestampMs: number) => {
-        const container = await makeContainer();
-        const defaultDataObject = await requestFluidObject<ITestDataObject>(container, "default");
-        await waitForContainerConnection(container);
-
-        const handleKey = "handle";
-        const dataStore = await defaultDataObject._context.containerRuntime.createDataStore(TestDataObjectType);
-        const testDataObject = await requestFluidObject<ITestDataObject>(dataStore, "");
-        const unreferencedId = testDataObject._context.id;
-
-        // Reference a datastore - important for making it live
-        defaultDataObject._root.set(handleKey, testDataObject.handle);
-        // Unreference a datastore
-        defaultDataObject._root.delete(handleKey);
-
-        // Summarize
-        const {
-            container: summarizingContainer1,
-            summarizer: summarizer1,
-        } = await loadSummarizerAndContainer();
-        const summaryVersion = (await summarize(summarizer1)).summaryVersion;
-
-        // TODO: trailing op test - note because of the way gc is currently structured, the error isn't logged,
-        // but it is detected - it's stored in the pending queue and the container closes before the error is sent.
-        testDataObject._root.set("send while unreferenced", "op");
-        await provider.ensureSynchronized();
-
-        // Close the containers as these containers would be closed by session expiry before sweep ready ever occurs
-        container.close();
-        summarizingContainer1.close();
-
-        // Wait some time, the datastore can be in many different unreference states
-        await delay(approximateUnreferenceTimestampMs);
-
-        // Load a new container and summarizer based on the latest summary, summarize
-        const {
-            container: summarizingContainer2,
-            summarizer: summarizer2,
-        } = await loadSummarizerAndContainer(summaryVersion);
-
-        return {
-            unreferencedId,
-            summarizingContainer: summarizingContainer2,
-            summarizer: summarizer2,
-            summaryVersion,
+    describe("Using tombstone data stores is not allowed", () => {
```

Comment:
Did you change my tombstone datastores pr at all? I would assume not, but the changes look crazy.

Reply:
I only moved them around and did a little restructuring. Disable whitespaces in the PR settings and then take a look:
<img width="398" alt="image" src="https://user-images.githubusercontent.com/45832642/201427169-08154985-9562-44c2-8a13-ba16ef25b1c0.png">

Reply:
That worked nicely :)

Reply:
Remember to turn it off though :-). It's helpful to see whitespace changes.